rpm-x64/Dockerfile

```
@@ -44,11 +49,18 @@ ENV RUSTC_SHA256 $RUSTC_SHA256
 ENV PERLBREW_VERSION $PERLBREW_VERSION
 ENV PERLBREW_SHA256 $PERLBREW_SHA256
 ENV PERL_VERSION $PERL_VERSION
+ENV BINUTILS_VERSION $BINUTILS_VERSION
+ENV BINUTILS_SHA256 $BINUTILS_SHA256
+ENV BASE_IMAGE $BASE_IMAGE
+
+# persist RHEL major for readable output
+RUN echo $(cat /etc/redhat-release | cut -d'.' -f1 | awk '{print $NF}') > /etc/redhat-release-major
 
 # Enable the vault (archive) repos, as we are past CentOS6 EOL
-RUN sed -is 's/enabled=0/enabled=1/g' /etc/yum.repos.d/CentOS-Vault.repo && \
+RUN if [[ $(cat /etc/redhat-release-major) == 6 ]]; then \ 
```

This check is reused 5+ times in the file. Can we set a variable/func and then check that instead?
```suggestion
RUN if [[ "$(cat /etc/redhat-release-major)" == 6 ]]; then \ 
```

---

nit: Do we want to print out some debug info if we end up in this block? If so - we may want to do that for other similar blocks.